src/query/procedure/mg_procedure_impl.cpp

```diff
@@ -2796,3 +2796,27 @@ mgp_error mgp_module_add_function(mgp_module *module, const char *name, mgp_func
       },
       result);
 }
+
+void mgp_log(const mgp_log_level log_level, const char *output) {
+  switch (log_level) {
+    case mgp_log_level::MGP_LOG_LEVEL_TRACE:
+      spdlog::trace(output);
+      return;
+    case mgp_log_level::MGP_LOG_LEVEL_DEBUG:
+      spdlog::debug(output);
+      return;
+    case mgp_log_level::MGP_LOG_LEVEL_INFO:
+      spdlog::info(output);
+      return;
+    case mgp_log_level::MGP_LOG_LEVEL_WARN:
+      spdlog::warn(output);
+      return;
+    case mgp_log_level::MGP_LOG_LEVEL_ERROR:
+      spdlog::error(output);
+      return;
+    case mgp_log_level::MGP_LOG_LEVEL_CRITICAL:
+      spdlog::critical(output);
+      return;
+  }
+  throw std::invalid_argument{fmt::format("Invalid log level: {}", log_level)};
```

Comment:
I just noticed this throw is wrong here. If you want to throw, then wrap the implementation of the function, as `mgp_log` is a C API function, it cannot let exceptions go over the API, as exceptions are not existing in C.